docs/plugins.md

```diff
@@ -153,6 +153,35 @@ in the cycle and paid out 1234.98 XTZ in rewards.
 #ourbakery #rewards #tezos
 ```
 
+## Discord Plugin
+
+This plugin makes an HTTP POST request to a Discord Webhook URL.
+
+You must be an admin of a discord server to generate a webhook URL. Do so by going to 'Server Settings' -> 'Integrations' -> 'New Webhook'. Select the name, upload an avatar icon (if desired), and select which channel the messages will appear.
+
+Paste the generated URL into the appropriate field in the plugin section. You can customize the message for 'payouts' notifications. This is handled exactly as in the twitter plygin
+
+### Example Config
+
+* endpoint: The webhook URL generated by your discord server
+* send_admin: Whether or not to send administrative-style messages using the bot. These messages will appear alongside general user-style messages.
```

Comment:
In general I would be a fan of defining the type like <bool> or <string> but maybe the example is enough... or false/true